go.mod

```diff
@@ -3,14 +3,15 @@ module github.com/jmattheis/goverter
 go 1.18
 
 require (
-	github.com/dave/jennifer v1.6.0
-	github.com/stretchr/testify v1.8.1
-	golang.org/x/tools v0.17.0
+	github.com/dave/jennifer v1.7.1
+	github.com/stretchr/testify v1.9.0
+	golang.org/x/tools v0.24.0
```

Comment:
go1.18 build is broken because of the dep update.

Reply:
Sorry, I thought I brought it up just _before_ 1.18 incompatibility and those checks weren't running for me to notice. Will fix.